targets/python/templates/Errors.py.ejs

```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+
+# Error codes returned by PlayFabAPIs
+class PlayFabErrorCode(Enum):
+    Success = 0,
+    Unknown = 1,
+    ConnectionError = 2,
+    JsonParseError = 3,
+    <% for(var i=1; i<errorList.length-1; i++) { var errorProps = errors[errorList[i]] %><%- errorProps.name %> = <%- errorProps.id %>,
+    <% } %><% var errorProps = errors[errorList[errorList.length-1]] %><%- errorProps.name %> = <%- errorProps.id %>
+
+class PlayFabError:
+
+    def __init__(self, *args):
```

Comment:
Can you document the intended type of args? It looks like you expect 1 json string, but it isn't totally clear to me.